styleguide/js/zoolander.js

```diff
@@ -21,14 +21,41 @@ jQuery(document).ready(function ($) {
     jQuery ('.title a').click(function (event) {
       jQuery (this).parent().parent().find('.fa-arrow-right').animate({  borderSpacing: +90 }, {
       step: function(now,fx) {
-          $(this).css('-webkit-transform','rotate('+now+'deg)'); 
+          $(this).css('-webkit-transform','rotate('+now+'deg)');
           $(this).css('-moz-transform','rotate('+now+'deg)');
           $(this).css('transform','rotate('+now+'deg)');
           counter = 1;
       },
       duration:'slow'
       },'linear');
     });
-});
-
 
+    mobileFooterCheck();
+    $(window).resize(function(){
+      mobileFooterCheck();
+    });
+    function mobileFooterCheck(){
+      var footerSectionId = 0;
+      $('.footer ul li span.menu-title').each(function(){
+        footerSectionId++;
+        var _this = $(this);
+        var attr = _this.attr('data-toggle');
+        //tells us if screen is tablet & or mobile
+        if ($('.footer ul li').css('text-align')=='center') {
```

Comment:
I'm not sure checking text alignment is the best way to identify viewport. If for whatever reason it ever becomes left-aligned we'd have to go back and fix this functionality. 

Perhaps something like this instead? (Note: This is not a definitive solution, but more of something open for discussion). 
http://stackoverflow.com/questions/18575582/how-to-detect-responsive-breakpoints-of-twitter-bootstrap-3-using-javascript